code/ui/blocks/src/blocks/Source.tsx

```diff
@@ -118,8 +118,7 @@ export const useSourceProps = (
       // Always fall back to the primary story for source parameters, even if code is set.
       stories = [docsContext.storyById()];
     } catch (err) {
-      // You are allowed to use <Story code="..." /> unattached.
-      if (!props.code) throw err;
```

Comment:
@tmeasday I had to remove this check to make story-less, unattached Canvases work.

Reply:
Fails gracefully 
Source:
![image](https://user-images.githubusercontent.com/5678122/215441225-4cc2a585-1486-4013-b8ff-cc6553dcb09f.png)
Canvas:
![image](https://user-images.githubusercontent.com/5678122/215441337-f88a9023-ce67-4e82-be13-1e9af069bfcb.png)